charts/lfx-platform/templates/mailpit/httproute.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright The Linux Foundation and each contributor to LFX.
+# SPDX-License-Identifier: MIT
+---
+{{ if .Values.mailpit.enabled -}}
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: mailpit
+  namespace: {{ .Release.Namespace }}
+spec:
+  parentRefs:
+    - name: {{ .Values.traefik.gateway.name }}
+      sectionName: {{ include "lfx-platform.default-listener" . }}
+      namespace: {{ .Release.Namespace }}
+  hostnames:
+    - 'mailpit.{{ .Values.lfx.domain }}'
+  rules:
+    - backendRefs:
+        - name: lfx-platform-mailpit-http
```

Comment:
See comments on Authelia -- lets match the correct service name as it's subject to templating

- https://github.com/jouve/charts/blob/main/charts/mailpit/templates/http-service.yaml#L4
- https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_names.tpl#L21-L38